src/main/scala/com/workday/warp/common/WarpPropertyManager.scala

```diff
@@ -46,9 +44,20 @@ object WarpPropertyManager {
   val propertyFile: String = computePropertyFile
 
   // load the configuration file
-  Try(configuration.read(new FileReader(propertyFile))) recover {
-    case exception: ConfigurationException => Logger.error(exception, s"Error loading WARP Configuration file: $propertyFile \n\n")
-  }
+  // note that if the file does not exist, we log a warning and continue execution with an empty property set.
+  // (all default property values will be used)
+  // if there is an unrecoverable exception from configuration library, we'll throw that exception
+  val configuration: PropertiesConfiguration = Try(new Configurations().properties(propertyFile)).recoverWith {
+    case _: ConfigurationException if !new File(propertyFile).exists() =>
+      Logger.warn(s"$propertyFile does not exist!" +
+        "\n    Be aware that if the property values you require have not been passed in as Java System properties" +
+        "\n    this program is very likely to fail when an unset required property is accessed.\n")
+      // fall back on empty config
+      Success(new PropertiesConfiguration)
+    case exception: Exception =>
```

Comment:
i dont think these 3 lines are particularly necessary. we're just logging an error and re-wrapping the exception. an exception, if there is one, will be thrown on line 60